conans/model/scm.py

```diff
@@ -1,8 +1,11 @@
 import json
 import os
 
+from future.moves import subprocess
```

Comment:
Why ``from future``? So far importing subprocess hasn't been an issue.

Reply:
IDE autocompleting, I'll fix it